scripts/validate-test-results

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env perl
+
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+use strict;
+use warnings;
+use experimental 'signatures';
+
+use Mojo::JSON qw(decode_json);
+use Mojo::Path;
+use Mojo::File qw(path);
+use Test::More;
+
+sub validate_test_result($result_file_path) {
```

Comment:
our usual styles suggest a space between the function name and the opening parenthesis. Which shows that apparently this file isn't covered by any style and linting checks hence I suggest to not have a perl script like this in this repo at all.  Keep in mind that os-autoinst-distri-example should be an example that is easily transferrable into other test distributions. Coming up with extra scripts makes that harder